### ISSUE TITLE
Make the navigation bar "dropdown" a button in a11x

### ISSUE THREAD

**Issue by BasThomas** [rnystrom/GitHawk#1404]
The newly introduced navigation bar dropdown does not have a button trait; it should!

See [this discussion](https://github.com/rnystrom/GitHawk/pull/1399#discussion_r161381804) in #1399.
